frontend/src/scenes/session-recordings/file-playback/SessionRecordingFilePlayback.tsx

```diff
@@ -17,6 +18,8 @@ export function SessionRecordingFilePlayback(): JSX.Element {
     const { hasAvailableFeature } = useValues(userLogic)
     const filePlaybackEnabled = hasAvailableFeature(AvailableFeature.RECORDINGS_FILE_EXPORT)
 
+    const dropRef = createRef<HTMLDivElement>()
```

Comment:
Pretty sure this should be a `useRef`, since we're in a function component

Reply:
and fixed where I copy-pasta-d from